### PR TITLE
Address unintiuitive behavior on Race/Ethnicity popup

### LINKED ISSUE #684: Address unintiuitive behavior on Race/Ethnicity popup
Currently, if you say NO to the question "Are you able to record an individual’s ethnicity (Hispanic or Latino, Not Hispanic or Latino, or Unknown Ethnicity) separately from their race in your case management system?", and then you try to turn off the Unknown Race category, the Hispanic or Latino Race category will also automatically turn off. Similarly, if both of those categories are turn off but you turn on the Hispanic or Latino Race category, the Unknown Race category will also turn on. Video:

https://images.zenhubusercontent.com/108306686/b380490f-8f3c-43e8-bdbc-a3372671ffea/screen_recording_2023_05_30_at_5_20_42_pm.mov

This looks like a bug, even though it isn't. The reason for this behavior is that Justice Counts considers Hispanic or Latino as an ethnicity, not a race. If an agency's case management system considers Hispanic or Latino as a race, not an ethnicity, then in order to convert that data to Justice Counts, we will record these individuals as Race: Unknown and Ethnicity: Hispanic or Latino. So, we need the Unknown Race category to be turned on if the Hispanic or Latino Race category is turned on, otherwise we can't do this conversion.

How we should "fix": If the Hispanic or Latino Race category is turned on, **then disable the Unknown Race category toggle** (i.e. "freeze" it in the On state). Add a little question mark tooltip next to it that says "The Justice Counts data model requires the Unknown Race category to be turned on if the Hispanic or Latino Race category is turned on." If the user turns off the Hispanic or Latino Race category, then they can turn off the Unknown Race category too.

### PR DESCRIPTION
**Pull request by ivan-kishko**
## Description of the change

> Disable unknown race toggle if Hispanic or Latino is turned on

## Type of change
Type: Feature

> All pull requests must have at least one of the following labels applied (otherwise the PR will fail):

| Label                       	| Description                                                                                               	|
|-----------------------------	|-----------------------------------------------------------------------------------------------------------	|
| Type: Bug                   	| non-breaking change that fixes an issue                                                                   	|
| Type: Feature               	| non-breaking change that adds functionality                                                               	|
| Type: Breaking Change       	| fix or feature that would cause existing functionality to not work as expected                            	|
| Type: Non-breaking refactor 	| change addresses some tech debt item or prepares for a later change, but does not change functionality    	|
| Type: Configuration Change  	| adjusts configuration to achieve some end related to functionality, development, performance, or security 	|
| Type: Dependency Upgrade      | upgrades a project dependency - these changes are not included in release notes                             |

## Related issues

Closes #684 

### Development

**This box MUST be checked by the submitter prior to merging**:
- [x] **Double- and triple-checked that there is no Personally Identifiable Information (PII) being mistakenly added in this pull request**

These boxes should be checked by the submitter prior to merging:
- [ ] Tests have been written to cover the code changed/added as part of this pull request

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by ivan-kishko** [comment]
@lilidworkin The only thing, we do not have question mark icon in our assets so I added info icon

**Comment by ivan-kishko** [comment]
> @ivan-kishko this looks so good and is exactly what I envisioned, thank you so much!!!
> 
> Tiny thing -- would you mind making the height of the dialog just a tad bigger? It looks a bit weird at first because you can only see one of the radio buttons.
> 
> @mxosman in case you're curious here's what it looks like -- wdyt of this solution?
> 
>  Screen.Recording.2023-05-31.at.11.13.40.AM.mov

@lilidworkin here it is [ed91ecf](https://github.com/Recidiviz/justice-counts/pull/685/commits/ed91ecf9ee79455d3619c7e69848c5afa9415376). Is this high enough?

File: publisher/src/components/MetricsConfiguration/RaceEthnicitiesModalForm.tsx [hunk 1]
```diff
@@ -183,23 +185,45 @@ function RaceEthnicitiesModalForm({
...
+                        src={infoIcon}
+                        alt=""
+                      />
+                      <Tooltip
+                        anchorId="anchor"
+                        place="right"
+                        content="The Justice Counts data model requires the Unknown Race category to be turned on if the Hispanic or Latino Race category is turned on."
+                        variant="dark"
+                        style={{ maxWidth: "300px" }}
+                      />
```
Comment (mxosman) [inline publisher/src/components/MetricsConfiguration/RaceEthnicitiesModalForm.tsx:221]: Nice!

**Comment by mxosman** [comment]
> @ivan-kishko this looks so good and is exactly what I envisioned, thank you so much!!!
> 
> Tiny thing -- would you mind making the height of the dialog just a tad bigger? It looks a bit weird at first because you can only see one of the radio buttons.
> 
> @mxosman in case you're curious here's what it looks like -- wdyt of this solution?

I LOVE it! This is such a fantastic solution!!! So simple yet clears up so much confusion. 

>"The Justice Counts data model requires the Unknown Race category to be turned on if the Hispanic or Latino Race category is turned on."

I almost wish you could add on something like ~"because this is where you will record those numbers" - but keep the same succinctness of the current copy.

File: publisher/src/components/MetricsConfiguration/RaceEthnicitiesModalForm.tsx [hunk 2]
```diff
@@ -15,6 +15,7 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 // =============================================================================
 
+import infoIcon from "@justice-counts/common/assets/info-icon.svg";
```
Comment (mxosman) [inline publisher/src/components/MetricsConfiguration/RaceEthnicitiesModalForm.tsx:18]: Can we import these as `ReactComponents` - SVGs embedded in images load slower?

**Review by mxosman — APPROVED** [review]

File: publisher/src/components/MetricsConfiguration/RaceEthnicitiesModalForm.tsx (hunk 2 above)
Comment (ivan-kishko) [reply publisher/src/components/MetricsConfiguration/RaceEthnicitiesModalForm.tsx]: [27ce148](https://github.com/Recidiviz/justice-counts/pull/685/commits/27ce1488be259adf3d6482290c8754edabf7d48b)

Comment (mxosman) [reply publisher/src/components/MetricsConfiguration/RaceEthnicitiesModalForm.tsx]: Thank you, Ivan!

**Pull request merged**